DOCUMENTATION.md

```diff
@@ -334,6 +334,16 @@ This is a special folder (`/tmp/pi-apps-local-packages`) used by the `install_pa
   - This is useful for the `createapp` script to automatically find a suitable icon for package-apps you're trying to add.
   - This uses `dpkg-query` to list all files each package installed. The list is filteres to only show `.png` files in `/icons/` or `/pixmaps/` folders.
   - The list is sorted by filesize to find the picture with the most pixels.
+- `ubuntu_ppa_installer` - Given a ppa name, this function will automatically add the apt repo and key for the current ubuntu dist to the users apt sources.list.d directory.
+ - Example usage:
+    ```bash
+    ubuntu_ppa_installer "theofficialgman/flatpak-no-bwrap"
+    ```
+- `debian_ppa_installer` - Given a ppa name, dist, and key this function will automatically add the apt repo and key for the specified ubuntu dist to the users apt sources.list.d directory.
```

Comment:
It may be worth mentioning the naming scheme for the sources.list.d file. Future script-writers should probably know how to remove a repository when an app is uninstalled.

Reply:
done